### PR TITLE
Indicate when built as a preview a of PR

### PR DESCRIPTION
**Pull request by 36degrees**
Use the build environment variables that Netlify give us to detect when a build happens as part of a PR preview, and clearly mark it as such by replacing the beta banner with a suitable summary.

screen shot 2018-01-18 at 16 09 59

https://trello.com/c/VzFEH5eR/549-review-apps-should-be-clearly-identified-as-previews

### CODE REVIEW

File: views/partials/_banner.njk [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{% from "phase-banner/macro.njk" import govukPhaseBanner %}
+
+{% if PULL_REQUEST %}
+  {# Extract the PR number from Netlify's DEPLOY_PRIME_URL, which looks like
```
Comment (36degrees) [inline views/partials/_banner.njk]: I've sent an email to Netlify to see if there's a nicer way to do this – or indeed if they could add a nicer way of doing this.

**Comment by 36degrees** [comment]
Turns out there is an undocumented `REVIEW_ID` env variable, which whilst far less fun is probably a more sensible way of doing things.

**Comment by nickcolley** [comment]
💀 RIP gnarly regex 👍

File: lib/metalsmith.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ const sass = require('metalsmith-sass')             // convert Sass files to CSS
 
 const debug = require('./debug')                    // debug plugin
 const navigation = require('./navigation.js')       // navigation plugin
+const env = require('metalsmith-env');              // environment vars plugin
```
Comment (alex-ju) [inline lib/metalsmith.js]: 😨 `;`

we should probably remove it at some point 😄

Comment (36degrees) [reply lib/metalsmith.js]: OH NOES

Comment (36degrees) [reply lib/metalsmith.js]: https://github.com/alphagov/govuk-design-system/pull/144

**Pull request merged**